src/Question/Serializers/QuestionNormalizer.php

```diff
@@ -60,6 +60,7 @@ function (Answer $answer) use ($format) {
             'imageFileName' => $question->getImageFileName()->toNative(),
             'answers' => $answers,
             'feedback' => $question->getFeedback()->toNative(),
+            'createdOn' => $question->getCreatedOn()->format('Y-m-d H:i:s')
```

Comment:
@stejes dit is zonder tijdzone, beter gebruik maken van `DateTime::ATOM` of gelijkaardige.